PythonToolbox/quantconnect/api.py

```diff
@@ -13,7 +13,7 @@
 
 from datetime import datetime
 from json import dumps
-from requests import Request
+from requests import get as requests_get, Request
```

Comment:
was there a naming collision here that required the alias? if not, my preference would be to not provide the alias as they tend to make code less readable as it requires one to determine what the alias means. That being said, the chosen alias is _extremely_ explicit, so the aforementioned statement really doesn't apply. I certainly like the naming convention of `<object>_<method>` for aliases as it's almost like it wasn't aliased at all 👍